package.json

```diff
@@ -18,6 +18,7 @@
     "debug": "*",
     "errorhandler": "~1.3.2",
     "express": "^4.12.0",
+    "handlebars": "^2.0.0",
```

Comment:
i don't really like the idea of adding handlebars as a requirement to rendr - i'd rather see this live at the persons app level, if this is a requirement for https://github.com/rendrjs/rendr-handlebars/pull/52 then i think we should probably actually leave it in rendr-handlebars to separate the concerns.

I was thinking that each app's package.json could define the version of handlebars they want to use if they're using `rendr-handlebars` and `rendr-handlebars` is simply the glue that can bind handlebars for rendr.